docs/source/getting_started/benchmarks/cifar10_benchmark.py

```diff
@@ -167,6 +169,11 @@
     gaussian_blur=0.0,
 )
 
+# Multi crop augmentation for FastSiam
+fast_siam_transform = MultiViewTransform(
+    [SimCLRViewTransform(input_size=32, cj_strength=0.5, gaussian_blur=0.0)] * 4
+)
```

Comment:
We could add a new fastsiam transform.

Reply:
I'm not sure this makes sense. It's just a lot of copy pasted code... But I can add it if you think it's a good idea.

Reply:
SimSiam uses SimCLR transform right? In that case we can just make an alias. Does FastSiam change any transform parameters compared to SimSiam?

Reply:
No they don't change anything compared to SimSiam. But it's possible SimSiam uses a different min_scale than SimCLR